terraform/modules/bulk_upload/access_log_bucket.tf

```diff
@@ -0,0 +1,72 @@
+#tfsec:ignore:aws-s3-enable-bucket-encryption: default bucket encryption is sufficient
+#tfsec:ignore:aws-s3-enable-bucket-logging: access log bucket doesn't need access logging itself
+#tfsec:ignore:aws-s3-encryption-customer-key: default encryption is sufficient
+#tfsec:ignore:aws-s3-enable-versioning: Not important, each log will be a new file with a different name

```

Comment:
Do we want versioning as a form of deletion protection?

Reply:
I think we do, implemented in this PR now 👍